values.yaml

```diff
@@ -60,3 +66,7094 @@ mdai-operator:
+                            "type": "prometheus",
+                            "uid": "$datasource"
+                          },
+                          "editorMode": "code",
+                          "exemplar": true,
+                          "expr": "sum(otelcol_otelsvc_k8s_pod_deleted${suffix}{job=\"$job\"}) by (job $grouping)",
+                          "format": "time_series",
+                          "hide": false,
+                          "interval": "$minstep",
+                          "intervalFactor": 1,
+                          "legendFormat": "Deleted: {{transport}} {{service_instance_id}}",
+                          "range": true,
+                          "refId": "C"
+                        }
+                      ],
+                      "title": "Pods",
+                      "type": "timeseries"
+                    }
+                  ],
+                  "title": "Kubernetes",
+                  "type": "row"
+                },
+                {
+                  "collapsed": false,
+                  "gridPos": {
+                    "h": 1,
+                    "w": 24,
+                    "x": 0,
+                    "y": 81
+                  },
+                  "id": 66,
+                  "panels": [],
+                  "title": "Documentation",
+                  "type": "row"
+                },
+                {
+                  "datasource": {
+                    "type": "prometheus",
+                    "uid": "$datasource"
+                  },
+                  "editable": true,
+                  "error": false,
+                  "gridPos": {
+                    "h": 3,
+                    "w": 24,
+                    "x": 0,
+                    "y": 82
+                  },
+                  "id": 45,
+                  "links": [],
+                  "options": {
+                    "code": {
+                      "language": "plaintext",
+                      "showLineNumbers": false,
+                      "showMiniMap": false
+                    },
+                    "content": "<a href=\"http://www.monitoringartist.com\" target=\"_blank\" title=\"Dashboard maintained by Monitoring Artist - DevOps / Docker / Kubernetes / AWS ECS / Google GCP / Zabbix / Zenoss / Terraform / Monitoring\"><img src=\"https://monitoringartist.github.io/monitoring-artist-logo-grafana.png\" height=\"30px\" /></a> | \n<a target=\"_blank\" href=\"https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md#metrics\">OTEL collector troubleshooting (how to enable telemetry metrics)</a> | \n<a target=\"_blank\" href=\"https://opentelemetry.io/docs/collector/scaling/\">Scaling the Collector (metrics to watch)</a> | \n<a target=\"_blank\" href=\"https://opentelemetry.io/docs/collector/internal-telemetry/\">Collector internal telemetry</a> | \n<a target=\"_blank\" href=\"https://www.otelbin.io/\">OTelBin (OpenTelemetry colle
```

Comment:
We should put all the dashboard values at the very bottom of this file, so no one has to ever scroll all the way down past these horrible JSON blobs.

Reply:
I wanted to put them into their own `.json` files and reference them